docs/get-started/installation.md

```diff
@@ -0,0 +1,159 @@
+  - 5.0.300 - C:\Program Files\dotnet\sdk\5.0.300
+  - 6.0.100-preview.4.21255.9 - C:\Program Files\dotnet\sdk\6.0.100-preview.4.21255.9
+
+> .NET SDK - EnableWorkloadResolver.sentinel Checkup...
+  - C:\Program Files (x86)\Microsoft Visual
+Studio\2019\Preview\MSBuild\Current\Bin\SdkResolvers\Microsoft.DotNet.MSBuildSdkResolver\EnableWorkloadResolver.sentinel
+exists.
+  - C:\Program Files (x86)\Microsoft Visual
+Studio\2019\Enterprise\MSBuild\Current\Bin\SdkResolvers\Microsoft.DotNet.MSBuildSdkResolver\EnableWorkloadResolver.senti
+nel exists.
+  - C:\Program Files\dotnet\sdk\6.0.100-preview.4.21255.9\EnableWorkloadResolver.sentinel exists.
+
+> .NET SDK - Workloads (6.0.100-preview.4.21255.9) Checkup...
+  - microsoft-android-sdk-full (Microsoft.NET.Workload.Android : 11.0.200-preview.4.245) installed.
+  - microsoft-ios-sdk-full (Microsoft.NET.Workload.iOS : 14.5.100-preview.4.638) installed.
+  - microsoft-maccatalyst-sdk-full (Microsoft.NET.Workload.MacCatalyst : 14.5.100-preview.4.638) installed.
+  - microsoft-tvos-sdk-full (Microsoft.NET.Workload.tvOS : 14.5.100-preview.4.638) installed.
+  - microsoft-macos-sdk-full (Microsoft.NET.Workload.macOS : 11.3.100-preview.4.638) installed.
+
+> .NET SDK - Packs (6.0.100-preview.4.21255.9) Checkup...
+  - Microsoft.Maui.Templates (6.0.100-preview.4.634) installed.
+  - Microsoft.iOS.Windows.Sdk (14.5.100-preview.4.638) installed.
+  - Microsoft.Android.Sdk (11.0.200-preview.4.245) installed.
+  - Microsoft.Android.Sdk.BundleTool (11.0.200-preview.4.245) installed.
+  - Microsoft.Android.Ref (11.0.200-preview.4.245) installed.
+  - Microsoft.Android.Templates (11.0.200-preview.4.245) installed.
+  - Microsoft.iOS.Sdk (14.5.100-preview.4.638) installed.
+  - Microsoft.iOS.Ref (14.5.100-preview.4.638) installed.
+  - Microsoft.iOS.Templates (14.5.100-preview.4.638) installed.
+  - Microsoft.MacCatalyst.Sdk (14.5.100-preview.4.638) installed.
+  - Microsoft.MacCatalyst.Ref (14.5.100-preview.4.638) installed.
+  - Microsoft.MacCatalyst.Templates (14.5.100-preview.4.638) installed.
+  - Microsoft.tvOS.Sdk (14.5.100-preview.4.638) installed.
+  - Microsoft.tvOS.Ref (14.5.100-preview.4.638) installed.
+  - Microsoft.tvOS.Templates (14.5.100-preview.4.638) installed.
+  - Microsoft.macOS.Sdk (11.3.100-preview.4.638) installed.
+  - Microsoft.macOS.Ref (11.3.100-preview.4.638) installed.
+  - Microsoft.macOS.Templates (11.3.100-preview.4.638) installed.
+────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+
+- Congratulations, everything looks great!
+```
+
+## Build and launch apps
+
+For instructions on how to create and build your first .NET MAUI app, see [Build your first .NET MAUI app](first-app.md).
+
+Once a .NET MAUI app has been created, it can be built using .NET CLI if required. The following .NET CLI commands show how to build and launch a .NET MAUI app on three different platforms:
```

Comment:
"if required" -- why would building with the CLI be "required"?

```suggestion
Once a .NET MAUI app has been created, it can be built using the .NET CLI if required. The following .NET CLI commands show how to build and launch a .NET MAUI app on three different platforms:
```

Reply:
This would be better expressed as a .NET MAUI app can be built using .NET CLI if the user chooses to (rather than using VS). I've fixed this in my branch.